File: SDKs/apideck.com/file-storage/8.85.0/python/src/sdk/models/shared/filessearch.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class FilesSearch:
    query: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('query') }})
    drive_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('drive_id') }})
    

File: SDKs/apideck.com/file-storage/8.85.0/typescript/src/sdk/models/shared/filessearch.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class FilesSearch extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=drive_id" })
  driveId?: string;

  @SpeakeasyMetadata({ data: "json, name=query" })
  query: string;
}
